test/mango.py

```diff
@@ -32,8 +33,7 @@ def __init__(self, host, port, dbname, auth=None):
         self.port = port
         self.dbname = dbname
         self.sess = requests.session()
-        if auth is not None:
-            self.sess.auth = auth
+        self.sess.auth = ('adm', 'pass')
```

Comment:
What credentials does travis use? In our mango Makefile, we run nosetests, but the tests will fail with wrong credentials if it does not match what couchdb is started with.